lib/web_ui/lib/ui_web/src/ui_web/images.dart

```diff
@@ -25,3 +28,18 @@ Future<ui.Codec> createImageCodecFromUrl(
     chunkCallback: chunkCallback,
   );
 }
+
+/// Creates a [ui.Image] from an ImageBitmap object.
```

Comment:
nit: dartdoc likes it when there's one empty line between the first and the second paragraph (or between paragraphs in general).